src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      {isMarried
```

Comment:
```suggestion
    <section className="Person">
      <h2 className="Person__name">{`My name is ${name}`}</h2>
      
      {age && <p className="Person__age">{`I am ${age}`}</p>}
      
      {isMarried
```
Add empty lines between sibling element to make your code easier to read